src/Valet/Commands/AzureDevOps/BuildPipeline/DryRun.cs

```diff
@@ -11,5 +11,14 @@ public DryRun(string[] args) : base(args)
 
     protected override string Name => "pipeline";
     protected override string Description => "Target a designer or YAML pipeline";
-    protected override ImmutableArray<Option> Options => ImmutableArray<Option>.Empty;
+
+    public static readonly Option<FileInfo> SourceFilePath = new("--source-file-path")
```

Comment:
Should there be a `--source-file-path` option for the `migrate` [subcommand](https://github.com/github/gh-valet/blob/main/src/Valet/Commands/AzureDevOps/BuildPipeline/Migrate.cs)?